components/browser/menu/src/main/java/mozilla/components/browser/menu/WebExtensionBrowserMenu.kt

```diff
@@ -80,6 +81,11 @@ class WebExtensionBrowserMenu internal constructor(
                         return@forEach
                     }
 
+                    if (extension.id == "webcompat-reporter@mozilla.org") {
```

Comment:
Hm this is the part I expected to happen in the application, not the component. It's the application that specifies / decided which menu item to promote. I don't think we should hard code that in the component.